public/app/features/provisioning/RecentJobs.tsx

```diff
@@ -120,14 +120,33 @@ interface ExpandedRowProps {
 function ExpandedRow({ row }: ExpandedRowProps) {
   const hasSummary = Boolean(row.status?.summary?.length);
   const hasErrors = Boolean(row.status?.errors?.length);
+  const hasSpec = Boolean(row.spec);
+  const specJson = hasSpec ? JSON.stringify(row.spec, null, 2) : '';
 
-  if (!hasSummary && !hasErrors) {
+  if (!hasSummary && !hasErrors && !hasSpec) {
+    console.log('no summary, errors, or spec', row);
     return null;
   }
 
   return (
     <Box padding={2}>
       <Stack direction="column" gap={2}>
+        {hasSpec && specJson && (
```

Comment:
```suggestion
        {specJson && (
```
You don't really need `hasSpec` here since `specJson` depends on it.